Laixer/FunderMaps
# v3.2.1

### Merge when:

* [ ] The merge build succeeded
* [ ] The PR has been reviewed and is approved
* [ ] All packages are upgraded to the latest version (only PR into master)

### What issue does this PR address?

* **FunderMaps/IntegrationTests**
  - More tests

* **FunderMaps/WebApi**
  - Check inquiry creator equals reviewer

* **FunderMaps/Data**
  - Remove unused organization fields
  - Apply navigation filters

### Does this PR introduce a breaking change?

Yes, organization DTO changed

### Additional information

* [x] Remove branch after merge (only features into develop)
